src/components/ProgressBarComponent.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+/* Prime React components */
+import 'primereact/resources/primereact.min.css';
+import 'primereact/resources/themes/omega/theme.css';
+import 'font-awesome/css/font-awesome.css';
+import { ProgressBar } from 'primereact/components/progressbar/ProgressBar';
+// import {Growl} from 'primereact/components/growl/Growl';
+
+class ProgressBarComponent extends React.Component {
+    props: any;
+    state = {value1: 0, value2: 50};
```

Comment:
lint

Reply:
I removed this component. using the prime react component